### DISCUSSION TITLE
리뷰 상세 정보 조회 API 명세

### DISCUSSION THREAD

**Discussion by ohseil** [team-reviewing/reviewing-backend#18] [API 명세]
# 리뷰 상세 정보 조회   

## 기본 정보   
```http
GET /reviewers/{reviewer-id}/reviews/{review-id} HTTP/1.1
Host: ${ HOSTNAME }
Authorization: Bearer ${ ACCESS_TOKEN }
```

### Header
|Name|Description|Required|
|-|-|-|
|Authorization|access token|O|

## Response

|Name|Type|Description|Required|
|-|-|-|-|
|id|Long|리뷰 id|O|
|reviewerId|Long|리뷰어 id|O|
|title|String|리뷰 제목|O|
|content|String|리뷰 본문|O|
|prUrl|String|리뷰 PR URL|O|
|status|String|리뷰 상태|O|
|expireDate|String|리뷰의 상태 만료 시간|O|

### Request
```http
GET hostname/reviewers/1/reviews/1
Authorization: Bearer access_token
```

### Response
```http
HTTP/1.1 OK
Content-Type: application/json;charset=UTF-8
{
   "id" : 1,
   "reviewerId" : 1,
   "title" : "제목",
   "content" : "본문",
   "prUrl" : "prUrl",
   "status" : "CREATED",
   "expireDate" : "2023-05-01 16시"
}
```

**Comment by leeuihyun** [comment]
status만 추가된 게 맞을까요 ?

**Reply by ohseil** [reply]
넵 기존 명세에서 status 하나만 추가됐습니다!
